diff --git a/pom.xml b/pom.xml
index 528aaa7..9f9a9b8 100644
--- a/pom.xml
+++ b/pom.xml
@@ -29,4 +29,17 @@
 		</dependency>
 	</dependencies>
 
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.3</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
 </project>
diff --git a/src/main/java/clusterListener/ClusterListener.java b/src/main/java/clusterListener/ClusterListener.java
index e952c59..5830dd2 100644
--- a/src/main/java/clusterListener/ClusterListener.java
+++ b/src/main/java/clusterListener/ClusterListener.java
@@ -9,13 +9,6 @@ import clusterMassages.Pong;
 
 public class ClusterListener extends UntypedActor {
 
-  Cluster cluster = Cluster.get(context().system());
-
-  {
-    cluster.subscribe(self(), MemberEvent.class, ClusterEvent.class);
-    System.err.println(cluster);
-  }
-
   @Override
   public void onReceive(Object msg) throws Exception {
     System.out.println("ClusterListener.onReceive(): Nachricht erhalten");
diff --git a/src/main/java/clusterMassages/Ping.java b/src/main/java/clusterMassages/Ping.java
index c723a2d..54c9be5 100644
--- a/src/main/java/clusterMassages/Ping.java
+++ b/src/main/java/clusterMassages/Ping.java
@@ -1,6 +1,10 @@
 package clusterMassages;
 
-public class Ping {
+import java.io.Serializable;
+
+import akka.routing.ConsistentHashingRouter.ConsistentHashable;
+
+public class Ping implements ConsistentHashable, Serializable {
   public static long seqId = 0l;
   public String msg;
   public long msgId;
@@ -23,4 +27,8 @@ public class Ping {
   public String toString() {
     return "Ping [msg=" + msg + ", msgId=" + msgId + "]";
   }
+
+  public Object consistentHashKey() {
+    return msg;
+  }
 }
diff --git a/src/main/java/clusterMassages/Pong.java b/src/main/java/clusterMassages/Pong.java
index 9be69ae..ae02ddc 100644
--- a/src/main/java/clusterMassages/Pong.java
+++ b/src/main/java/clusterMassages/Pong.java
@@ -1,6 +1,8 @@
 package clusterMassages;
 
-public class Pong {
+import java.io.Serializable;
+
+public class Pong implements Serializable {
   public long msgId;
   public String message;
 
diff --git a/src/main/java/clusterSender/ClusterSender.java b/src/main/java/clusterSender/ClusterSender.java
index 87266ad..5c47cd7 100644
--- a/src/main/java/clusterSender/ClusterSender.java
+++ b/src/main/java/clusterSender/ClusterSender.java
@@ -13,14 +13,7 @@ import clusterMassages.Pong;
 
 public class ClusterSender extends UntypedActor {
   ActorRef listener = getContext()
-      .actorOf(FromConfig.getInstance().props(Props.create(ClusterListener.class)), "listener");
-
-  Cluster cluster = Cluster.get(context().system());
-
-  {
-    cluster.subscribe(self(), MemberEvent.class, ClusterEvent.class);
-    System.err.println(cluster);
-  }
+      .actorOf(FromConfig.getInstance().props(Props.create(ClusterListener.class)), "listenerRouter");
 
   @Override
   public void onReceive(Object message) throws Exception {
diff --git a/src/main/resources/application.conf b/src/main/resources/application.conf
index 1633bce..9dc961a 100644
--- a/src/main/resources/application.conf
+++ b/src/main/resources/application.conf
@@ -1,13 +1,5 @@
 akka {
 
-  loggers = ["akka.event.slf4j.Slf4jLogger"]
-  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
-  loglevel = "INFO"
-  stdout-loglevel = "INFO"
-  log-remote-lifecycle-events = on
-  log-dead-letters = on
-
-  log-dead-letters = off
 
   actor {
     provider = "akka.cluster.ClusterActorRefProvider"
@@ -22,7 +14,7 @@ akka {
   }
 
   cluster {
-    seed-nodes = ["akka.tcp://ClusterSystem@10.99.99.101:2551"]
+    seed-nodes = ["akka.tcp://ClusterSystem@127.0.0.1:2552"]
 
     auto-down-unreachable-after = 10s
 
diff --git a/src/main/resources/listener.conf b/src/main/resources/listener.conf
index af2e980..21d981d 100644
--- a/src/main/resources/listener.conf
+++ b/src/main/resources/listener.conf
@@ -5,7 +5,6 @@ akka.cluster.roles=[listener]
 akka{
 
   remote {
-    log-remote-lifecycle-events = off
     netty.tcp {
       hostname = 127.0.0.1
       port = 2553
diff --git a/src/main/resources/sender.conf b/src/main/resources/sender.conf
index 6053327..72603a5 100644
--- a/src/main/resources/sender.conf
+++ b/src/main/resources/sender.conf
@@ -5,7 +5,6 @@ akka.cluster.roles=[sender]
 akka{
 
   remote {
-    log-remote-lifecycle-events = off
     netty.tcp {
       hostname = 127.0.0.1
       port = 2552
@@ -14,7 +13,7 @@ akka{
   
   actor{
     deployment {
-      /sender = {
+      /sender/listenerRouter = {
         router = consistent-hashing-group
         routees.paths = ["/user/listener"]
         cluster {
